CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# How to contribute
+
+## Getting Started
+
+* Make sure you have a [GitHub account](https://github.com/signup/free)
+* Submit a ticket for your issue, assuming one does not already exist.
+  * Clearly describe the issue including steps to reproduce when it is a bug.
+
+## Making patches
+
+* Read [Manual (How to write packages for Nix)](https://nixos.org/nixpkgs/manual/)
+* Fork the repository on GitHub.
+* Create a branch for your future fix.
+  * You can make branch from a commit of your local `nixos-version`. That will help you to avoid additional local compilations. Because you will recieve some packages from cache server.
```

Comment:
Or use `$(git rev-parse --show-cdup)maintainers/scripts/update-channel-branches.sh` to pull latest versions of the channel as branches.